travis-scripts/ma-write.golden

```diff
@@ -0,0 +1,13 @@
+NAME:
```

Comment:
It took me a second to understand the naming of these files. I suggest renaming to something like `ma-read-output.golden` and `ma-write-help-text.golden`, so that it is understandable why are they not symmetric.